docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -211,4 +253,5 @@ PUT _cluster/settings
 
 You can use the <<cluster-remote-info, remote cluster info API>> to retrieve
 information about the configured remote clusters, as well as the remote nodes
-that the node is connected to.
+that the node is connected to. This is particularly useful to verify which
```

Comment:
> as well as the remote nodes that the node is connected to.

The API reference information (https://www.elastic.co/guide/en/elasticsearch/reference/master/cluster-remote-info.html) implies that "connected" is true/false.  Is that correct or does it list the gateway nodes?

Reply:
very good point. we do return a list of nodes but it's the list of seed nodes apparently rather than the selected gateway nodes, I need to adapt this then. Thanks for noticing!